setup.py

```diff
@@ -4,11 +4,12 @@
 
 Run tests in transactions using pytest, Flask, and SQLAlchemy.
 '''
-import os
 from setuptools import setup
 
-with open(os.path.join('requirements', 'main.txt')) as reqfile:
-    requirements = reqfile.read().splitlines()
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
```

Comment:
You got a great README. if you do this, it will be rendered nicely on pypi.org when you publish there.

Reply:
Great point!